gitpod-io/gitpod
# [gp] Make 'gp open <file>' work in Code as well

Fixes https://github.com/gitpod-io/gitpod/issues/3309

### How to test

1. https://jx-gp-open-code.staging.gitpod-dev.com/#https://github.com/gitpod-io/sveltejs-template
2. `gp open package.json`

Should work in both Theia and Code.
